playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -6,7 +6,7 @@
 # The AWS access/secret keys should be the keys of a separate user (not your main user), containing only the necessary S3 access role.
 # The 'clusterid' is the short name of your cluster.
 
-- hosts: tag_env-host-type_{{ clusterid }}-openshift-master
+- hosts: tag_environment_{{ env }}:&tag_host-type_openshift-master
```

Comment:
I would think that this would still use clusterid for now.

In other words, I think this should be:
```
- hosts: tag_environment_{{ clusterid }}:&tag_host-type_openshift-master
```